ipcalc.c

```diff
@@ -546,6 +547,7 @@ char *ipv4_prefix_to_hosts(char *hosts, unsigned hosts_size, unsigned prefix)
 	return hosts;
 }
 
+extern
```

Comment:
I believe the addition of extern here is superfluous. That's the default for C functions. The static keyword that you have added, on the other hand is the right thing to do. Would it make sense to keep the static changes in this commit?

Reply:
I like code to be explicit what is visibility of functions, but sure not a big deal. Updated commit no longer has externs here and there.